src/components/Home/ClusterDashboardItem.js

```diff
@@ -251,6 +254,22 @@ function ClusterDashboardItem({
               </span>
             </RefreshableLabel>
             <Dot style={{ paddingLeft: 0 }} />
+            <RefreshableLabel value={cluster.kubernetes_version}>
```

Comment:
there is no `kubernetes_version` field in clusters

Reply:
Right, changing